DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/CourseSetupController.cs

```diff
@@ -22,19 +32,31 @@
     public class CourseSetupController : Controller
     {
         private const string CourseFilterCookieName = "CourseFilter";
+        public const string SelectAllDiagnosticAction = "diagnostic-select-all";
+        public const string DeselectAllDiagnosticAction = "diagnostic-deselect-all";
+        public const string SelectAllLearningAction = "learning-select-all";
+        public const string DeselectAllLearningAction = "learning-deselect-all";

```

Comment:
I left them there because I make calls to them in the _EditSectionLearning and _EditSectionDiagnostic partials, e.g. 
`var selectAllAction = Model is EditCourseSectionViewModel ?
    CourseContentController.SelectAllLearningAction : CourseSetupController.SelectAllLearningAction;
  var deselectAllAction = Model is EditCourseSectionViewModel ?
    CourseContentController.DeselectAllLearningAction : CourseSetupController.DeselectAllLearningAction;`
but I've removed these consts from the Helper file and referenced the ones in the controller instead for concision

Reply:
What you've done has left 2 version of the constants still, some in CourseSetupController and some in CourseContentController. These need to be in one place, and since they are used by stuff in the helper, that is where they belong.

I don't see the point of the code you're referencing in the partials, the 2 strings are the same, and both controllers call the helper to process them anyway. If we were to update the string in one controller the other would break.

Reply:
Ah I see what you mean, moved them to the helper

Reply:
This still isn't quite resolved, the constants still exist in CourseContentController, any usages of those need replacing with the ones from the helper

Reply:
Ah sorry, all fixed now